yarn.lock

```diff
@@ -1409,6 +1409,24 @@
     debug "^3.1.0"
     glob "^10.4.2"
 
+"@expo/metro@~0.1.0":
```

Comment:
looks like we need to remove old versions of metro

Reply:
I'll follow this up after we're bumping to 0.81 👍